src/main/java/reposense/RepoSense.java

```diff
@@ -78,10 +79,8 @@ public static void main(String[] args) {
             if (cliArguments.isAutomaticallyLaunching()) {
                 ReportServer.startServer(SERVER_PORT_NUMBER, cliArguments.getOutputFilePath().toAbsolutePath());
             }
-        } catch (IOException ioe) {
+        } catch (IOException | ParseException | InvalidCsvException ioe) {
```

Comment:
The name of the exception should `e` as it can stand for multiple exceptions now.

Reply:
I missed this. Done.